guides/source/command_line.md

```diff
@@ -376,37 +376,37 @@ $ bin/rails destroy model Oops
       remove      test/fixtures/oops.yml
 ```
 
-Rake
-----
+bin/rails
+---------
 
-Rake is Ruby Make, a standalone Ruby utility that replaces the Unix utility 'make', and uses a 'Rakefile' and `.rake` files to build up a list of tasks. In Rails, Rake is used for common administration tasks, especially sophisticated ones that build off of each other.
+Rake is Ruby Make, a standalone Ruby utility that replaces the Unix utility 'make', and uses a 'Rakefile' and `.rake` files to build up a list of tasks. In Rails, Rake is used for common administration tasks, especially sophisticated ones that build off of each other. Since Rails 5.0+ rake commands implemented in Rails Framework itself. We prefer to use bin/rails instead of rake.
```

Comment:
Change "Since Rails 5.0+ rake commands implemented in Rails Framework itself. We prefer to use bin/rails instead of rake."

to: "Since Rails 5.0+ has rake commands built into the `rails executable`, `bin/rails` is the new default for running commands.